### PR TITLE
[WIP] Adding collection of docker and openshift versions

### PR DESCRIPTION
**Pull request by alongoldboim**
This pr counts on having ops-metrics-client, which will send metric to Hawkular and Zabbix.

### CODE REVIEW

File: scripts/monitoring/cron-send-rpms-versions.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# Check OpenShift version
+OC_VERSION = oc version | sed -n '1{p;q}' | sed 's/[^0-9.]*\([0-9.]*\).*/\1/'
+
+# Check Docker version
+DOCKER_VERSION = docker --version | sed 's/[^0-9.]*\([0-9.]*\).*/\1/'
+
+HOSTNAME = hostname --fqdn
+
+# Send results to zabbix
+ops-metrics-client -k "$HOSTNAME.docker-version" -o "$DOCKER_VERSION"
```
Comment (yaacov) [inline scripts/monitoring/cron-send-rpms-versions.sh]: The host name is added by the `ops-metrics-client` utility, for this line, the key `ops-metrics-client` creates will be: `/machine/$HOSTNAME/$HOSTNAME.docker-version`.
I think this line can be just:
`ops-metrics-client -k "docker.version" -o "$DOCKER_VERSION"`
and let the utility add the hostname.

File: scripts/monitoring/cron-send-rpms-versions.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+ops-metrics-client -k "$HOSTNAME.oc-version" -o "$OC_VERSION"
```
Comment (yaacov) [inline scripts/monitoring/cron-send-rpms-versions.sh]: I think the ops team like to use '.' when they separate words so the key is `docker.version` and not `docker-version`

This is important because we want to parse the key and use it to add missing tags (if they are not explicitly specified)
if you use `.` (ops notation) or `/` (heapster notation) we can parse it to tags `group=>'docker', metric=>'version'` , if you use `-` it will be   `group=>'docker-version', metric=>''` this is also ok ... since we do not know what to do with tags yet ...

File: scripts/monitoring/cron-send-rpms-versions.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Check OpenShift version
+OC_VERSION = oc version | sed -n '1{p;q}' | sed 's/[^0-9.]*\([0-9.]*\).*/\1/'
```
Comment (yaacov) [inline scripts/monitoring/cron-send-rpms-versions.sh]: `OC_VERSION=$(oc version | sed -n '1{p;q}' | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')` ?

Comment (yaacov) [reply scripts/monitoring/cron-send-rpms-versions.sh]: a. bash do not like space before `=`
b. bash will not return the return value of an expression unless you wrap it with `$(...)` or a tag

**Comment by alongoldboim** [comment]
@zgalor have a look.

**Comment by zgalor** [comment]
depends on https://github.com/openshift/openshift-tools/pull/1650

File: scripts/monitoring/cron-send-rpms-versions.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Check OpenShift version
+OC_VERSION=$(oc version | sed -n '1{p;q}' | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
+
+# Check Docker version
+DOCKER_VERSION=$(docker --version | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
+
+HOSTNAME = hostname --fqdn
```
Comment (zgalor) [inline scripts/monitoring/cron-send-rpms-versions.sh]: Why is that required?

Comment (alongoldboim) [reply scripts/monitoring/cron-send-rpms-versions.sh]: This pr isn't final, the actual key will be "machine/$HOSTNAME/docker.version", thats why i'm collecting it, will add a fix soon.

**Comment by alongoldboim** [comment]
@zgalor closing pr, take it over :)
